CONTRIBUTING.md

```diff
@@ -110,20 +110,22 @@ Later on, the original package-lock.json is restored when `git checkout package-
 
 To make a release, you first need to ensure that the released version will either be a semver minor or patch release so that automatic updates are working for our users. Following that, the process is simple:
 
-- Update `CHANGELOG.md` so that the unreleased section gets its version number. Commit and push this change.
 - Acquire an OTP token for 2fa.
 - Ensure you are logged in with npm in your shell (`npm whoami`)
-- Run either
-    - `NPM_CONFIG_OTP={your token} lerna publish --force-publish="*" patch && lerna bootstrap`, or
-    - `NPM_CONFIG_OTP={your token} lerna publish --force-publish="*" minor && lerna bootstrap`.
+- Run `NPM_CONFIG_OTP={your token} lerna publish && lerna bootstrap`.
+
+Lerna will determine if this is going to be a major, minor or patch version from the commit comments of all commits since the last release. It will also automatically update the CHANGELOG.md files in the root of the repository and also in all individual packages, based on those commits.
+
+If you want to review the changes that lerna is about to apply (for example the CHANGELOG files), you can execute `lerna version --no-git-tag-version --no-push` first.
```

Comment:
I would like to do this in two steps: Let's publish the next 2 or 3 releases still manually and if we encounter no issues there, we can switch to having it done automatically via the Github action (see below). Thus, for now, only the instructions on how to do a release are updated.